src/sentry/monitors/endpoints/organization_monitor_index.py

```diff
@@ -227,6 +228,11 @@ def post(self, request: Request, organization) -> Response:
         except MonitorLimitsExceeded as e:
             return self.respond({type(e).__name__: str(e)}, status=403)
 
+        # Attempt to assign a seat for this monitor
+        seat_outcome = quotas.backend.assign_monitor_seat(monitor)
```

Comment:
Curious question: Does this also apply to self-hosted customers, i.e can self-hosted customers create monitors without assigning a monitor seat?

Reply:
In self hosted the quotas system will accept everything.